### PR TITLE
Fixes #417 - Incorporated TopSky 2.5 beta 4

### LINKED ISSUE #417: TopSky 2.5
## What is the current situation?
TopSky 2.5 in beta, investigate compatibility with our settings

### PR DESCRIPTION
**Pull request by hazzas-99**
Fixes #417

# Summary of changes

- TopSky 2.5 beta 4
- Updated settings for compatibility
- Updated symbols for compatibility
- Updated CPDLC Unicom message
- Updated (fake) Eurocontrol CPDLC codes
- Updated various MinStack window displays on TopSky ASRs
- Updated default TopSky positions for CPDLC, CARD, Shortcut, and Sep windows
- Amended CARD settings - turned off Risk and Future by default

### CODE REVIEW

**Review by SamLefevre — APPROVED** [review]

**Review by Robbo599 — APPROVED** [review]

File: UK/Data/ASR/AC_C/C3T.asr [hunk 1]
```diff
@@ -1,6 +1,8 @@
 DisplayTypeName:Standard ES radar screen
 DisplayTypeNeedRadarContent:1
 DisplayTypeGeoReferenced:1
+SECTORFILE:
+SECTORTITLE:
```
Comment (luke11brown) [inline UK/Data/ASR/AC_C/C3T.asr:5]: Remove pls

File: UK/Data/Plugin/TopSky/TopSkyCPDLC.txt [hunk 2]
```diff
@@ -1,8 +1,8 @@
 [DCL]
-DCL:*:SID:<adep2> PDC <number> . . . . . CLD <hour><min> <day><month3><year2> <adep2> <callsign><endheader> CLRD TO <ades> OFF <drwy> VIA  <sid> ALT <cfl> SQUAWK <assr>. <qnh>HPA. <nfreq>. REPORT ACFT TYPE AND STAND NUMBER ON <freq> WHEN FULLY READY.
+DCL:*:SID:<adep2> PDC <number> . . . . . CLD <hour><min> <day><month3><year2> <adep2> <callsign><endheader> CLRD TO <ades> OFF <drwy> VIA <sid> ALT <cfl> SQUAWK <assr>. <qnh>HPA. <nfreq>. REPORT ACFT TYPE AND STAND NUMBER ON <freq> WHEN FULLY READY.
 
 [FREETEXT]
-FREETEXT:R:SERVICE TERMINATED. MONITOR UNICOM 122.800
+FREETEXT:UNICOM:SERVICE TERMINATED. MONITOR UNICOM 122.800
```
Comment (luke11brown) [inline UK/Data/Plugin/TopSky/TopSkyCPDLC.txt:5]: What's the effect on the aircraft side? As far as I understood the :R: provides them with an option to respond "ROGER" to the request. What happens now this has been changed? I imagine TopSky won't like the return of "UNICOM" in response to the sent message. Unable to test myself at the moment.

File: UK/Data/Plugin/TopSky/TopSkySoundSTCA.wav
Comment (luke11brown) [inline UK/Data/Plugin/TopSky/TopSkySoundSTCA.wav:1]: Which sound file is this? Is this the non-hospital sounding one?

**Comment by luke11brown** [comment]
Would like to ideally incorporate the eAUP provided by Prtugal vACC to provide danger area activation data based on real world Eurocontrol data feed. from [eAUP](https://eaup.vatsim.pt/api/docs). TopSky now accepts the data from the API. Only tasks from our side may require renaming our danger areas so they match the API nomenclature.

**Comment by luke11brown** [comment]
Quick brainstorm to complete the above:

- [ ] Set `HTTP_Areas_Remote_URL` in TopSkySettings.txt as `https://eaup.vatsim.pt/api/v2/areas/EG/`
- [ ] Create lines for all danger areas `AreaName:AUP:rsa_id` (`rsa_id` is to match string returned by eAUP)

**Comment by luke11brown** [comment]
image
image

**Pull request merged**